README.md

```diff
@@ -155,6 +155,65 @@ There are a total of six possible `timeTypes` to choose from.
 'queueWaitRange'         // Returns a Queue's current wait in minutes in the form of a range (this requires the the queueId to be passed)
 ```
 
+### Formatting and Custom Response Handling
+
+In some cases, you may wish to apply additional logic to a value, for example
+to display a custom message when the clinic is closed or to express the wait
+time as a range different from how you would show it on a Clockwise page.  In
+order to accomplish this, you can define a custom callback function, which will
+receive the value exactly as it is returned from the Clockwise API and should
+return the content that you wish to inject into the specified element on the
+page.
+
+Below is an example that displays a custom message any time the Clockwise API
+returns `"Closed"` and displays a message including a 10-minute range
```

Comment:
Kind of nit-picky, but the last half of this sentence is worded a little funny.  Maybe "returns `"Closed"`, otherwise it displays a 10-minute range" or something along those lines

Reply:
Yeah, I had a hard time with this.  We need to express "it turns a range in Clockwise's control into a range in your control" and "you're responsible for the whole sentence".  I guess that the note later on rally covers the "whole sentence" aspect though, so the tighter wording _does_ work better here.